test/services/oauth2/mock.js

```diff
@@ -14,7 +16,7 @@ chai.use(chaiHttp);
 
 const { expect } = chai;
 
-describe('oauth2 service mock', function oauthTest() {
+describe.skip('oauth2 service mock', function oauthTest() {
```

Comment:
Are you sure?

Reply:
actually, no. 
according to logic, the endpoint is used only from legacy client, where we have a feature flag
but there is at least another usage, according to grafana